xen/drivers/passthrough/arm/ipmmu-vmsa.c

```diff
@@ -329,11 +342,11 @@ static void set_archdata(struct device *dev, struct ipmmu_vmsa_archdata *p)
 #else
 static struct ipmmu_vmsa_archdata *to_archdata(struct device *dev)
```

Comment:
Not relevant to this patch. What is more to_archdata was directly ported from Linux